assets/js/edd-checkout-global.js

```diff
@@ -184,9 +184,11 @@ window.EDD_Checkout = (function($) {
 
 						} else {
 
-							if (!inputs.is('.card-address-2')) {
-								inputs.attr('required','required');
-							}
+							inputs.each( function ( input ) {
```

Comment:
```suggestion
							inputs.each( function ( index, input ) {
```

[The first parameter is the index in the array.](https://api.jquery.com/each/) The second one is the actual value. This was throwing a JavaScript error when testing with taxes + address fields enabled.

Reply:
Another option is to omit `index, input` completely and use `$( this )` inside of the callback to reference the current jQuery object directly vs. recreating it with `$( input )`.